### PR TITLE
Makefile: make bootloader naming consistent

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

**Review by x1ddos — APPROVED** [review]
Man, I was just thinking about renaming these myself the other day but then I was afraid you would be mad at me so I never sent a pull request :smile:

**Comment by benma** [comment]
> Man, I was just thinking about renaming these myself the other day but then I was afraid you would be mad at me so I never sent a pull request smile

uuh why would i be mad? you didn't know hate inconsistent naming? :joy:

File: .ci/ci [hunk 1]
```diff
@@ -32,16 +32,16 @@ make coverage
 
 # Bootloader variants
 make -j8 bootloader
-make -j8 bootloader-devdevice
-make -j8 bootloader-devdevice-locked
+make -j8 bootloader-development
+make -j8 bootloader-development-locked
```
Comment (My1) [inline .ci/ci:36]: kinda curious what's a locked devdevice supposed to do? I mean isnt the point of a dev device to be unlocked so newer FW versions can be flashed onto it?

Comment (benma) [reply .ci/ci:36]: Locked does not refer to the signature checks on the firmware, but to MCU settings that prevent overwriting the flash with JLink.

Comment (My1) [reply .ci/ci:36]: oh okay interesting to know, thanks

**Pull request merged**